### PR TITLE
MB-9711 update load testing docs

### PR DESCRIPTION
**Pull request by pearl-truss**
## Description

As I'm going through the various load testing docs, I made some updates the readme in this repo to remove references to experimental

## Reviewer Notes

Are there any lingering references to experimental that need to be removed? Was there anything cut out in this PR that needs to be added back?

Is this still relevant?

> It is strongly recommended that you set up your `User` classes to
subclass `MilMoveHostMixin` so that your TLS settings are automatically updated when you switch from "local" to "exp."

## Setup

If the diff is hard to read, the updated README can be viewed [here](https://github.com/transcom/milmove_load_testing/blob/pm-mb-update-load-testing-docs/README.md)

## Code Review Verification Steps

* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-9711) for this change.

### CODE REVIEW

**Review by felipe-lee — COMMENTED** [review]
Thanks for making these updates! I have a few comments. Also since you removed one of the sections, the table of contents will need to be regenerated (`make generate_readme_toc`), though it'll mess up the spacing unfortunately so you'll probably have to tweak the indent level yourself.

File: README.md [hunk 1]
```diff
@@ -702,51 +702,35 @@ these steps. Otherwise, please follow the instructions in the `mymove` repo to c
...
-To load test against the AWS Experimental Environment you must modify the
-[`DEVLOCAL_AUTH` environment variable](https://github.com/transcom/mymove/blob/master/config/env/exp.app.env#L8)
-and [deploy the code to the experimental environment](https://github.com/transcom/mymove/wiki/deploy-to-experimental).
+```sh
+ waf_ip_rate_limit   = 20000
+```
 
-Then, if you have `User` classes that take advantage of the `MilMoveHostMixin` class, you may run your load tests using
-"dp3" as the host value. If not, make sure to use the loadtest domains as your host:
+### Metrics
```
[the remark this answers is not in the document]
Comment (pearl-truss) [reply README.md:716]: Thanks, I removed the content of this section and replaced it with a link to those docs. Let me know if that works :)

[the remark this answers is not in the document]
Comment (felipe-lee) [reply README.md:716]: I think the wrong link accidentally got put in. The one right now is about making the house smell like christmas (I do appreciate the link though 😃 )

[the remark this answers is not in the document]
Comment (pearl-truss) [reply README.md:716]: oh gosh :doh:.. This should be fixed now :)

**Review by felipe-lee — APPROVED** [review]
Awesome, thanks for updating these docs!

**Review by duncan-truss — APPROVED** [review]

**Pull request merged**